python/ray/tests/test_advanced_9.py

```diff
@@ -420,7 +420,7 @@ def test_omp_threads_set_third_party(ray_start_cluster, monkeypatch):
         cluster.add_node(num_cpus=4)
         ray.init(address=cluster.address)
 
-        @ray.remote(num_cpus=2)
+        @ray.remote(num_cpus=1)
```

Comment:
This is the reason that it broke the test I guess? 

We use `num_cpus` for `OMP_NUM_THREADS` for a task - why did it get changed?

Reply:
Thanks for your review, this change is for test only, and it won't affect the result since I tested before.

This PR imports `pyarrow`, yet I cannot get the relationship with the break.. Could you find where the `4` comes from in the break assertation?

Reply:
likely the 4 comes from the `num_cpus` passed into init.

Reply:
Looks like it's fixed on the current HEAD?

Reply:
Yes, I found it a little bit tricky since it's fixed after I moved `import pyarrow` into functions..

Reply:
Some other cases failed, and I have no idea whether theses are related..

Reply:
you could ignore the metric heads test - these should be fixed in master btw. 

The arrow test failure looks relevant?

Reply:
Got it , just found `WorkerCrashedError` in Arrow 6, yet no details.